trillian.proto

```diff
@@ -39,6 +39,11 @@ enum HashStrategy {
   // Certificate Transparency strategy: leaf hash prefix = 0x00, node prefix =
   // 0x01, empty hash is digest([]byte{}), as defined in the specification.
   RFC_6962 = 1;
+
+  // Sparse Merkle Tree strategy:  leaf hash prefix = 0x00, node prefix = 0x01,
+  // empty branch is resursivily computed from empty leaf nodes. 
+  // NOT secure in a multi tree environment. For testing only.
```

Comment:
It think it should probably be explicitly called "TEST_MAP_HASHER" here and elsewhere and reserve MAP_HASHER for the secure version.